rb/ketama.py

```diff
@@ -5,7 +5,7 @@
 
 
 def md5_bytes(key):
-    return map(ord, hashlib.md5(key).digest())
+    return bytearray(hashlib.md5(key.encode("utf-8")).digest())
```

Comment:
Isn't `.encode('ansi')` is better here?

Reply:
The key could be unicode no?

Reply:
And if it is just an ascii string it will encode to the same value, since unicode is bc with ascii encoding.

Reply:
I should have been clearer lol, I meant instead of the `bytearray()` call, not replacing the encoding. Anyway, I think this is cool.